src/workerd/jsg/modules.c++

```diff
@@ -433,7 +432,7 @@ v8::Local<v8::Module> createSyntheticModule(
   return v8::Module::CreateSyntheticModule(
       js.v8Isolate,
       v8StrIntern(js.v8Isolate, name),
-      exportNames,
```

Comment:
Since we no longer need to pass a std::vector to the V8 interface, have you considered eliminating it here or replacing it with a KJ equivalent?

Reply:
yeah, we probably can. Will do that separately tho.